fastddsspy_tool/test/application/dds/AdvancedConfigurationExample/arg_configuration.h

```diff
@@ -229,6 +230,8 @@ const option::Descriptor usage[] = {
       "  -p <str> \t--partitions=<str> \tPartitions to match separated by ';'."
       " Single or double quotes required with multiple partitions."
       " With empty string ('') no partitions used. (Default: '')." },
+    { PROFILE, 0, "x", "xml-profile",         Arg::String,
+      "  -x <str> \t--xml-profile=<str> \tProfile name to configure DomainParticipant."},
```

Comment:
Extra space missing.
```suggestion
      "  -x <str> \t--xml-profile=<str>  \tProfile name to configure DomainParticipant."},
```

Reply:
Not needed